sdk/eventhub/azure-eventhub/samples/async_samples/client_creation_async.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+Examples to show how to create async EventHubProducerClient/EventHubConsumerClient
+"""
+import asyncio
+import os
+from azure.eventhub import TransportType
+from azure.eventhub.aio import EventHubProducerClient, EventHubConsumerClient, EventHubSharedKeyCredential
+
+
+connection_string = os.environ['EVENT_HUB_CONN_STR']
+fully_qualified_namespace = os.environ['EVENT_HUB_HOSTNAME']
+eventhub_name = os.environ['EVENT_HUB_NAME']
+sas_policy = os.environ['EVENT_HUB_SAS_POLICY']
+sas_key = os.environ['EVENT_HUB_SAS_KEY']
+consumer_group = "$Default"
+
+
+async def create_producer_client():
+    print('Examples showing how to create producer client.')
+
+    # Create producer client from connection string
+
+    producer_client = EventHubProducerClient.from_connection_string(
+        conn_str=connection_string  # connection string contains EventHub name
+    )
+
+    # Illustration of commonly used parameters
+    producer_client = EventHubProducerClient.from_connection_string(
+        conn_str=connection_string,
+        logging_enable=True,  # Enable log for network tracing
+        eventhub_name=eventhub_name,  # EventHub name should be specified if it doesn't show up in connection string
+        retry_total=3,  # Retry up to 3 times to re-do failed operations
+        transport_type=TransportType.AmqpOverWebsocket  # Use websocket as the underlying transport protocol
```

Comment:
Suggest use the default value amqp because most users don't need web socket. When they copy the code, they may copy this part and don't change it. This sample is not to demo web socket.

Reply:
Similary logging_enable=False. 
Is it better to put eventhub_name before logging_enable? conn_str and eventhub_name should be together.

Reply:
updated

Reply:
updated